parallel-consumer-examples/parallel-consumer-example-core/src/main/java/io/confluent/parallelconsumer/examples/core/CoreApp.java

```diff
@@ -141,6 +141,9 @@ void maxRetries() {
         final int maxRetries = 10;
         final Map<ConsumerRecord<String, String>, Long> retriesCount = new ConcurrentHashMap<>();
 
+        pc.pollAndProduceMany();
+        pc.poll(records ->);
+
```

Comment:
presumably - to be removed?

Reply:
yeah I was playing with what the api ended up looking like